Dockerfile

```diff
@@ -24,26 +27,38 @@ RUN apt-get update -qq && \
   libffi-dev \
   libpng-dev \
   libpq-dev \
-  curl && \
-  apt-get clean && \
-  rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
-  mkdir /install
-
-WORKDIR /install
+  curl
 
-# Copy as early as possible so we can cache ...
+# Copy only what we really need
+COPY README.md .
+COPY setup.py .
 COPY requirements.txt .
 
-RUN pip install -r requirements.txt --no-cache-dir
+# Install Rasa and its dependencies
+RUN pip install -r requirements.txt
+
+# Install Rasa as package
+COPY rasa ./rasa
+RUN pip install -e .
```

Comment:
given that this is a production container, I don't think we should install in development mode `-e` in the first place - can you please check what best practices are regarding this

Reply:
done, simply using `pip install .` now